src/cmdstan-guide/pathfinder_intro.Rmd

```diff
@@ -0,0 +1,144 @@
+              5      -6.748e+00      9.936e-05   3.738e-07    1.000e+00  1.000e+00       126 -6.164e+00 -6.164e+00                  
+Path [2] :Best Iter: [5] ELBO (-6.164015) evaluations: (126)
+Path [3] :Initial log joint density = -18.986308
+Path [3] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes 
+              5      -6.748e+00      2.996e-04   4.018e-06    1.000e+00  1.000e+00       126 -6.201e+00 -6.201e+00                  
+Path [3] :Best Iter: [5] ELBO (-6.200559) evaluations: (126)
+Path [4] :Initial log joint density = -8.304453
+Path [4] : Iter      log prob        ||dx||      ||grad||     alpha      alpha0      # evals       ELBO    Best ELBO        Notes 
+              5      -6.748e+00      2.814e-04   2.034e-06    1.000e+00  1.000e+00       126 -6.221e+00 -6.221e+00                  
+Path [4] :Best Iter: [3] ELBO (-6.161276) evaluations: (126)
+Total log probability function evaluations:8404
+```
+
+Pathfinder output is written into a set of StanCSV files; one file which contains the importance resampled draws
+from multi-path Pathfinder, and one file which contains the output of each single-path Pathfinder run.
+
+Given an output file name, CmdStan adds suffixes to the base filename to distinguish between the output files.
+For the default output file name `output.csv`, the resulting set of CSV files are
+```
+output_pathfinder.csv
+output_1.csv
+output_2.csv
+output_3.csv
+output_4.csv
+```
+
+The file `output_pathfinder.csv` contains the importance-resampled draws; the other output files contain the draws from the single-path Pathfinder runs.   For both sets of outputs, the organization of information is the same.
+The initial CSV comment rows contain the complete set of CmdStan configuration options.
+Next is the column header line, followed the set of approximate draws.
+
+The pathfinder algorithm first outputs `lp_approx__`, the log density in the approximating distribution, and `lp__`,  the log density in the target distribution, followed by estimates of the model parameters, transformed parameters, and generated quantities.
+```
+lp_approx__,lp__,theta
+-2.4973, -8.2951, 0.0811852
+-0.87445, -7.06526, 0.160207
+-0.812285, -7.07124, 0.35819
+
+...
+```
+
+The final lines are comment lines which give timing information.
+```
+# 
+# Elapsed Time: 0.016000 seconds (Pathfinders)
+#               0.003000 seconds (PSIS)
+#               0.019000 seconds (Total)
+# 
+```
+
+The contents of the individual pathfinders have the same structure, except that the timing information is just the timing information for the single pathfinder run.
+```
+# 
+# Elapsed Time: 0.003000 seconds (Pathfinder)
+# 
+```
+
+For further details, see
+Zhang, Lu, Bob Carpenter, Andrew Gelman, and Aki Vehtari. 2022.
+_Pathfinder: Parallel quasi-Newton variational inference._
+arxiv 2108.03782. https://doi.org/10.48550/arXiv.2108.03782
```

Comment:
I think we should add these citations to the `bibtex/all.bib` file and use them that way.

We also might want to consider using the official JMLR version, rather than the arxiv? It's still free to read, https://www.jmlr.org/papers/v23/21-0889.html